log/formatters.go

```diff
@@ -23,27 +26,77 @@ type jsonFormatter struct {
 	timestampColumn string
 }
 
+// Defining a log line type so that we can use a much faster JSON marshall-ing package
+type logData struct {
+	level        string
+	timestamp    string
+	timestampKey string
```

Comment:
so while this works, it might be a bit confusing, maybe define timestamp as its own struct due to the ability to customize the "timestamp" key text